lib/services/auth/bloc/auth_cubit.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:mynotes/services/auth/bloc/auth_state.dart';
+import 'package:mynotes/services/auth/firebase_auth_provider.dart';
+
+class AuthCubit extends Cubit<AuthState> {
```

Comment:
let's introduce dependency injection by annotation, but let's introduce it in seprate PR